src/components/Admin/AdminActive.vue

```diff
@@ -36,6 +36,47 @@
 
           <v-list-item-subtitle v-text="'Return Date: ' + device.returnDate" />
         </v-list-item-content>
+
+        <v-list-item-action>
+          <template>
+            <v-btn
+              icon
+              color="grey lighten-1"
+              @click="dialog = !dialog"
+            >
+              <v-icon>mdi-information</v-icon>
+            </v-btn>
+          </template>
+        </v-list-item-action>
+
+        <v-dialog
+          v-model="dialog"
+          max-width="500px"
```

Comment:
```suggestion
          max-width="500px"
          :retain-focus="false"
```

Reply:
That should fix it. test it and if it works just commit the change and I'll approve. Good work!